ix-dev/community/windmill/templates/docker-compose.yaml

```diff
@@ -0,0 +1,176 @@
+
+{% do c_server.environment.add_env("DATABASE_URL", database_url) %}
+{% do c_server.environment.add_env("MODE", "server") %}
+{# TZ is already defined automatically from the library #}
+{% do c_server.environment.add_user_envs(values.windmill.additional_envs) %}
+
+{% do c_server.depends.add_dependency(values.consts.postgres_container_name, "service_healthy") %}
+
+{% do c_server.add_storage("/tmp/windmill/logs", values.storage.worker_logs) %}
+{% do perm_container.add_or_skip_action("worker_logs", values.storage.worker_logs, {"uid": values.run_as.user, "gid": values.run_as.group, "mode": "check"}) %}
+
+{% do c_server.expose.add_port(8000) %}
+{% do c_server.expose.add_port(2525) %}
+
+{# Windmill Worker Container #}
+{% set c_worker = tpl.add_container(values.consts.windmill_worker_container_name, "image") %}
+{% do c_worker.set_user(values.run_as.user, values.run_as.group) %}
+{% do c_worker.healthcheck.set_test("wget", {"port": 8000, "path": "/api/health", "scheme": "http"}) %}
+
+{% do c_worker.environment.add_env("DATABASE_URL", database_url) %}
+{% do c_worker.environment.add_env("MODE", "worker") %}
+{% do c_worker.environment.add_env("WORKER_GROUP", "default") %}
+{# TZ is already defined automatically from the library #}
+{% do c_worker.environment.add_user_envs(values.windmill.additional_envs) %}
+
+{% do c_worker.depends.add_dependency(values.consts.postgres_container_name, "service_healthy") %}
+
+{% do c_worker.add_storage("/tmp/windmill/cache", values.storage.worker_cache) %}
+{% do perm_container.add_or_skip_action("worker_cache", values.storage.worker_cache, {"uid": values.run_as.user, "gid": values.run_as.group, "mode": "check"}) %}
+
+{% do c_worker.add_storage("/tmp/windmill/logs", values.storage.worker_logs) %}
+
+{% if values.worker.mount_docker_socket %}
+{% do c_worker.add_storage("/var/run/docker.sock", {"type": "host_path", "host_path_config": {"path": "/var/run/docker.sock"}}) %}
+{% endif %}
+
+{# Commented out due to missing resources attribute
+{% do c_worker.resources.set_limits({"cpu": values.resources.limits.worker_cpus, "memory": values.resources.limits.worker_memory}) %}
+#}
+{# Commented out due to missing scale attribute
+{% do c_worker.scale.set_replicas(values.worker.replicas) %}
+#}
+
+{# Windmill Worker Native Container #}
+{% set c_worker_native = tpl.add_container(values.consts.windmill_worker_native_container_name, "image") %}
+{% do c_worker_native.set_user(values.run_as.user, values.run_as.group) %}
+{% do c_worker_native.healthcheck.set_test("wget", {"port": 8000, "path": "/api/health", "scheme": "http"}) %}
+
+{% do c_worker_native.environment.add_env("DATABASE_URL", database_url) %}
+{% do c_worker_native.environment.add_env("MODE", "worker") %}
+{% do c_worker_native.environment.add_env("WORKER_GROUP", "native") %}
+{% do c_worker_native.environment.add_env("NUM_WORKERS", values.worker.native_num_workers) %}
+{% do c_worker_native.environment.add_env("SLEEP_QUEUE", "200") %}
```

Comment:
you can use numbers directly. everything will be converted to string automatically